dictionaries/pt/filters.yml

```diff
@@ -1,31 +1,31 @@
 pt:
-  edit_sumamry: 'relatórios %{num} filtros editar obsoletos'
+  edit_summary: 'Informando %{num} filtros de edição obsoletos'
   last_changed: 'Última [[Especial:Filtro de abusos/history/%{id}|alteração]]'
-  summary: '%{num} filtros editar ativado sem visitas nos últimos %{days} dias'
-  title: 'Filtros editar obsoletos como de %{date}'
-  'abuseFilter': 'Filtro de abusos'
+  summary: '%{num} filtros de edição ativados sem ocorrências nos últimos %{days} dias'
+  title: 'Filtros de edição obsoletos em %{date}'
+  'abuseFilter': 'Filtro de edições'
   'actions': 'Ações'
   'at': 'em'
   'by': 'por'
   'description': 'Descrição'
   'disallow': 'não autorizar'
-  'edit filters': 'filtros editar'
-  'failed to write to template': 'Falha ao gravar no modelo'
+  'edit filters': 'filtros de edição'
+  'failed to write to template': 'Falha ao gravar na predefinição'
   'fatal error': 'Erro fatal'
   'filter': 'Filtro'
-  'enabled filters with no hits in %{num}': 'filtros ativados sem visitas em'
+  'enabled filters with no hits in %{num}': 'filtros ativados sem ocorrências em'
   'flags': 'Sinalizações'
```

Comment:
@MusikAnimal : Maybe we can use `marcas` (meaning `marks`)? Or in case these four options are the only flags, `marcado como $1` (meaning `marked as $1`), where `$1` would be the list of `marks` for the filter.